app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  width: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &:focus,
+  &:hover {
+    color: white;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  }
+  &:active {
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.15);
+  }
+  @media screen and (max-width: 600px) {
+    display: none;
+  }
```

Comment:
We probably don't want this, right? It feels strange to me to hide a button via the button's own styles through a media query on the page width, since I would normally think that only "layout-level" components should really be doing stuff like that. It doesn't really matter in this case because we already switch to the hamburger menu at <= 600px, but I think this would get in the way if we wanted to reuse these styles in other contexts.

Reply:
I think this was leftover from some tinkering, and I forgot to remove it when I got the higher-level responsiveness worked out